hermes.podspec

```diff
@@ -22,14 +24,14 @@ end
 
 Pod::Spec.new do |spec|
   spec.name        = "hermes"
-  spec.version     = "0.5.1"
+  spec.version     = "0.5.2-rc.1"
```

Comment:
Let's either revert this change and make another commit (like https://github.com/facebook/hermes/commit/49c2875605e190f9d8fdf66c72a30bbce5db6739) to bump all version occurrences to 0.5.3 for consistency, or I think it would be also nice to just bump all versions occurrences to 0.5.3 directly in this PR (and we can directly tag this as `v0.5.3`).

Reply:
Cool, will update 👍

Reply:
Done 👍